huxley/core/middlewares.py

```diff
@@ -1,35 +1,46 @@
-# Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
+# Copyright (c) 2011-2017 Berkeley Model United Nations. All rights reserved.
 # Use of this source code is governed by a BSD License (see LICENSE).
 
 import json, logging, sys, traceback
 
 
 class ExceptionLoggerMiddleware(object):
 
-    def process_exception(self, request, exception):
-        logger = logging.getLogger('huxley.exception')
-        exc_type, exc_value, exc_traceback = sys.exc_info()
-        exc_traceback = ''.join(traceback.format_exception(exc_type, exc_value, exc_traceback))
-        log = json.dumps({
-              'message': exc_traceback,
-              'uri': request.path,
-              'status_code': 500,
-              'username': request.user.username})
-        logger.exception(log)
+  def __init__(self, get_response):
+    self.get_response = get_response
```

Comment:
Middleware factories must accept a `get_response object`, and I think `__init__` is the means by which class based middleware factories accept it. [Link](https://docs.djangoproject.com/en/1.11/topics/http/middleware/#init-get-response)